### ISSUE TITLE
Show "links not implemented" only once

### ISSUE THREAD

**Issue by Tim-Erwin** [beauby/jsonapi-datastore#31]
I think it would be enough to only warn once about links being not yet implemented. Right now it kinda spams the console.

**Comment by Chapabu** [comment]
I'm using this module as part of a Daemon, so it would be nice to have an option to not show this message at all as well. Perhaps it could read `NODE_ENV` and only show it on non-production environments?

Thinking about it, does it even need to show at all? Would a note in the README suffice?

**Comment by ucarbehlul** [comment]
What does it even mean? Can it be replaced with something more clear?

**Comment by Archelyst** [comment]
@ucarbehlul , see https://jsonapi.org/format/#document-links for the meaning of "links". This feature is not supported, i.e. the "links" property is ignored.
